ack/readers/awin_advertiser/reader.py

```diff
@@ -65,6 +70,10 @@ def request(self):
             build_url, params=payload
         )
         json_response = response.json()
+        # if report is not a campaign report, remove tags from the response
+        if self.remove_tags == 'True' and self.report_type != REPORT_TYPES[2]:
```

Comment:
If you make remove_tags a boolean you just have to put:
`if self.remove_tags and self.report_type != REPORT_TYPES[2]:`

Reply:
done